client/my-sites/plans-grid/components/actions.tsx

```diff
@@ -119,9 +117,23 @@ const SignupFlowPlanFeatureActionButton = ( {
 	}
 
 	return (
-		<Button className={ classes } onClick={ handleUpgradeButtonClick } busy={ busy }>
-			{ btnText }
-		</Button>
+		<div className="plan-features-2023-grid__actions-button-container">
+			<Button
+				className={ classes }
+				onClick={ () => handleUpgradeButtonClick( false ) }
+				busy={ busy }
+			>
+				{ btnText }
+			</Button>
+			{ hasFreeTrialPlan && (
+				<Button
+					onClick={ () => handleUpgradeButtonClick( true ) }
+					className="plan-features-2023-grid__actions-trial-button"
+				>
+					{ translate( 'Try for free' ) }
```

Comment:
Looks like we're introducing a new string. These either need to have translations completed before deploy, or the change should be hidden for non-EN users.

<img width="840" alt="Screenshot 2023-10-10 at 1 06 57 PM" src="https://github.com/Automattic/wp-calypso/assets/1269602/3abfa4cc-2fdf-4d91-95f3-9d51d07b0dd8">

Reply:
We don't need to rush. The PR is behind a feature flag and the release date is scheduled for a month from now.

Reply:
Sounds good, thanks for clarifying.